src/org.hdfgroup.hdfview/hdf/view/TableView/DefaultCompoundDSTableView.java

```diff
@@ -223,37 +237,33 @@ public Object getSelectedData() {
         }
         log.trace("getSelectedData(): size={} cName={} nt={}", size, cName, nt);
 
-        if (isStdRef) {
-            // std. ref data are stored in bytess
+        if (dataObject.getDatatype().isRef()) {
+            // ref data are stored in bytes
             selectedData = new byte[size];
```

Comment:
Comparing to DefaultScalarDSTableView, this allocates a `size`-sized byte array for the ref. data, whereas DefaultScalarDSTableView allocates a `size * (int)dataObject.getDatatype().getDatatypeSize()`-sized byte array. Should they be identical or does the DefaultCompoundDSTableView account for this?

Reply:
Right, compound seems to work differently. Although I didn't debug different choices for size.